make/configure.py

```diff
@@ -1399,7 +1399,10 @@ def createCLI( cross = None ):
     arch.mode.cli_add_argument( grp, '--arch' )
     grp.add_argument( '--cross', default=None, action='store', metavar='SPEC',
         help='specify GCC cross-compilation spec' )
-    cli.add_argument_group( grp )
+    if not (build_tuple.match('*-*-mingw*')):
+            grp.add_argument( '--enable-hardening', dest="enable_host_harden", default=None, action='store_true',
+                help='enable buffer overflow protection' )
+            cli.add_argument_group( grp )
```

Comment:
This should be on the same level as the if block, else the above arguments such as `--cross` will be hidden on other platforms.

Reply:
Thanks for your feedback. You mean the indentation needs to be on the same level as the if block? Or the lines have to come in a different order?